src/main/python/hydra/zmqtest/zmq_config.py

```diff
@@ -0,0 +1,6 @@
+# PUB server
+msg_count=10000  # Total messages to send
+msg_delay=0.001   # Delay between each message
```

Comment:
instead of delay if we can get a rate in bytes/sec as input it will be much better.